### PR TITLE
Add apache-benchmark, cerberus and kraken task

### PR DESCRIPTION
**Pull request by startxfr**
# Changes

- Add apache benchmark simple task to stress test an http/https endpoint
- Add kraken scenario task to use a chaos scenarios task in pipelines
- Add cerberus task to check for a cerberus positive signal
- Improve ab and cerberus script after linting requirements

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Follows the [authoring recommendations](https://github.com/tektoncd/catalog/blob/main/recommendations.md)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality of task changed or new task added)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [x] Complies with [Catalog Organization TEP][TEP], see [example]. **Note** [An issue has been filed to automate this validation][validation]
  - [x] File path follows  `<kind>/<name>/<version>/name.yaml`
  - [x] Has `README.md` at `<kind>/<name>/<version>/README.md`
  - [x] Has mandatory `metadata.labels` - `app.kubernetes.io/version` the same as the `<version>` of the resource
  - [x] Has mandatory `metadata.annotations` `tekton.dev/pipelines.minVersion`
  - [x] mandatory `spec.description` follows the convention

---

[TEP]: https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md
[example]: https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1
[validation]:  https://github.com/tektoncd/catalog/issues/413

### CODE REVIEW

**Comment by startxfr** [comment]
I've filled the CLA Authorization for both corporate and personal, but still showing a missing authorization. Could you help me ?

**Comment by vinamra28** [comment]
> I've filled the CLA Authorization for both corporate and personal, but still showing a missing authorization. Could you help me ?

@startxfr I guess it's fixed now. Also, as you have added the OWNERS file, are you willing to maintain these tasks in future? If so, can you please add yourself into the org? Probably you can refer the PR https://github.com/tektoncd/community/pull/680

File: task/kraken-scenario/0.1/kraken-scenario.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    kraken chaos testing is a good way to execute chaos testing of openshift or kubernetes
+    cluster as well as application running on top of it. This task allow you to use kraken
+    chaos scenarios as part of your application testing pipeline. It can also be part of
+    a complete chaos pipeline for your infrastructure layer.
+  params:
+    - name: awsCredentialSecret
+      type: string
+      description: Name of the secret key holding the aws-credentials (mandatory but used only for aws infrastructure chaos test).
+      default: kraken-aws-creds
```
[the remark this answers is not in the document]
Comment (startx-lab) [reply task/kraken-scenario/0.1/kraken-scenario.yaml:28]: Cloud provider support is optionnal and only use with some chaos scenarios related to the infrastructure. 
Kraken support GCP and Azure, but my task currently only support the AWS.

**Comment by startxfr** [comment]
I saw an XXL flag on this PR. Does it meed you prefer to split this PR into 3 in order to process the 3 task separately ?

**Comment by vdemeester** [comment]
> I saw an XXL flag on this PR. Does it meed you prefer to split this PR into 3 in order to process the 3 task separately ?

It would probably be slightly better yes 👍🏼

**Comment by startxfr** [comment]
> > I saw an XXL flag on this PR. Does it meed you prefer to split this PR into 3 in order to process the 3 task separately ?
> 
> It would probably be slightly better yes 👍🏼

ok, I remove this PR and create 3 other one.

**Pull request closed without merging**